OctaneVSPlugin/View/ResourceDictionary.xaml

```diff
@@ -282,5 +283,26 @@
             </Trigger>
         </Style.Triggers>
     </Style>
+    <Style x:Key="CheckComboBoxStyle1" TargetType="{x:Type wpfx:CheckComboBox}">
```

Comment:
I don't think this ended being used anywhere - I know I wrote it, you can delete it.